Sections/DDC.cs

```diff
@@ -101,15 +102,15 @@ public DDC(DahlDesign dahlDesign)
             Base.AddProp("SW1Brake",  0);
             Base.AddProp("SW1Throttle",  0);
 
-            Base.AddProp("DDUstartLED",  Base.Settings.DDUstartLED);
-            Base.AddProp("SW1startLED", Base.Settings.SW1startLED);
-            Base.AddProp("DDUEnabled",  Base.Settings.DDUEnabled);
+            Base.AttachDelegate("DDUstartLED", () => Base.Settings.DDUstartLED);
+            Base.AttachDelegate("SW1startLED", () => Base.Settings.SW1startLED);
+            Base.AttachDelegate("DDUEnabled", () => Base.Settings.DDUEnabled);
         }
 
         public void DataUpdate()
         {
-            Base.SetProp("DDCDDSEnabled",  false);
```

Comment:
So I think this breaks the DDS integration with the plugin menu. It is always false and I cant control the plugin menu with a DDS (Dahl Design Switch, a special kind of firmware for an encoder available in DDC)

Reply:
I compared this to some older commit.

The property DDCDDSEnabled was implicitly set to false on each iteration and later set to the value of Settings.DDSEnabled.

Is there any reason for this? I will now simply remove the implicit setting of the value DDSEnabled, it should be sufficiant that we have the binding from the value to the property using the delegate and also the connection from the settings page (XAML) to the property.

DDC.cs:
`Base.AttachDelegate("DDCDDSEnabled",  () => Base.Settings.DDSEnabled);`

will remove this (and also for DDCclutchEnabled, same issue here)
`Base.Settings.DDSEnabled = false;`

SettingsControl.xaml:
` <TextBlock Grid.Row="2" Text="DDS Enabled" />
                                <styles:SHToggleButton Grid.Row="2" Grid.Column="1" HorizontalAlignment="Left" VerticalAlignment="Top" Height="20" IsChecked="{Binding DDSEnabled}"/>
                               `

Reply:
I can't really say why I did that. It was one of the first settings I set up with a toggle in the menu. I might have had some misinterpretation of how the system works

Reply:
@andreasdahl1987 just let me know if it works now, then we can resolve this point